docs/source/development-guide/style-guide/naming-conventions.rst

```diff
@@ -31,46 +30,49 @@ The general filename convention is as follows::
   * SWE: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3``
 
 * ``<descriptor>`` stores information specific to the instrument and can store any information that is relevant. For
-  example, it will typically contain the data product name, and additionally may contain the pointing number (e.g.
-  ``burst-7`` for 'burst' mode and the 7th repointing). This field can have any text or numbers as long as it doesn't
-  include underscores. The following is a list of expected descriptors for each instrument:
+  example, it will typically contain the data product name, the sensor name, the timespan of the data, etc. This field
+  can have any text or numbers as long as it doesn't include underscores. The following is a list of expected
+  descriptors for each instrument:
 
   * CoDICE: ``hskp``, ``eng``, ``lo-priority``, ``lo-sw-angular``, ``lo-nsw-angular``, ``lo-pha``, ``hi-pha``,
     ``lo-sw-species``, ``lo-nsw-species``, ``hi-omni``, ``hi-sectored``
-  * GLOWS: ``histogram``, ``de``, ``<pointing number>``
+  * GLOWS: ``histogram``, ``de``
   * HIT: TBD
   * IDEX: TBD
-  * IMAP-Hi: TBD
+  * IMAP-Hi: ``45sensor-counts``, ``90sensor-counts``, ``45sensor-flux-3month-map`` (more TBD)
   * IMAP-Lo: TBD
-  * IMAP-Ultra: ``<pointing number>``
+  * IMAP-Ultra: ``45sensor-counts``, ``90sensor-counts`` (more TBD)
   * MAG: ``normal``, ``burst``
   * SWAPI: TBD
   * SWE: TBD
 
 * ``<startdate>`` is the date of the 'earliest' data within the data product, in the format of ``YYYYMMDD``.
-* ``<enddate>`` is the date of the 'latest' data within the data product if the data product spans more than one day.
-  If the data does not span multiple days, then the ``<enddate>`` will be the same as the ``<startdate>``. The format
-  is also ``YYYYMMDD``.
-* ``<version>`` stores the version of the data product in the format is ``vXX-YY``.  See the versioning conventions
-  described in the :ref:`data product versioning <Data Product Versioning>` documentation for further details.
+* ``<repointing>`` is an optional component that describes the repointing number, in the format of ``repoint-<num>``,
```

Comment:
I think we probably want to remove the "-" between repoint and number so it is a single repoint field and the hyphen separates the date and repointing fields.
```suggestion
* ``<repointing>`` is an optional component that describes the repointing number, in the format of ``repoint<num>``,
```

Reply:
Ah yes, good catch.